cmd/adapter-rest/startcmd/start.go

```diff
@@ -892,7 +892,7 @@ func addIssuerHandlers(parameters *adapterRestParameters, framework *aries.Aries
 	if err != nil {
 		return fmt.Errorf("failed to init trustbloc did creator: %w", err)
 	}
-	// TODO #572 Pass the output descriptors to issuer
+	// TODO #575 Persist the manifest output descriptor in database
```

Comment:
correction on TODO: it shouldn't be in database since you pass it everytime you start adapter, it should be in some kind of local file. based or in-memory cache.

add that new issue into current epic you are working on.

I don't see any issues in adapter numbered #575

Reply:
updated and fixed the issue # as well